stdlib/public/SwiftShims/LibcShims.h

```diff
@@ -93,22 +93,38 @@ _swift_stdlib_cxx11_mt19937_uniform(__swift_uint32_t upper_bound);
 // Math library functions
 static inline SWIFT_ALWAYS_INLINE
 float _swift_stdlib_remainderf(float _self, float _other) {
+#if __has_builtin(__builtin_remainderf)
   return __builtin_remainderf(_self, _other);
+#else
+  return remainderf(_self, _other);
```

Comment:
It must be a header being pulled in transitively. I actually think we should use 'sqrt' instead of '__builtin_sqrt' etc, as they compile to the same thing (in release builds at least) but I'm holding off for Greg's comments